src/components/Uploader/index.js

```diff
@@ -99,7 +99,7 @@ class Uploader extends Component {
       instructions
     } = this.props
     const isPoA = !!poaDocumentType
-    const documentTypeGroup = getDocumentTypeGroup(poaDocumentType || documentType)
+    const iconType = getDocumentTypeGroup(poaDocumentType || documentType) || 'identity'
```

Comment:
I wouldn't change the to `iconType` cause is not the actual info in the variable. This `${ camelCase(documentTypeGroup) }Icon` would be the value of a potential `iconType` variable from my understanding.
Also, couldn't you move the default value `identity` inside the helper  `getDocumentTypeGroup`?

Reply:
I changed the name from `iconType` to `uploadType`. Check this comment on why I changed `identity` to `face` https://github.com/onfido/onfido-sdk-ui/pull/807/files/19259b6e37be3b54a30af5ede559db238ca3a681#r331025231